flocker/provision/_install.py

```diff
@@ -141,17 +141,38 @@ def task_enable_docker():
     ]
 
 
-def task_disable_firewall():
+def configure_firewalld(rule):
```

Comment:
The firewalls rules aren't active by default, so we need to turn them on. The configuration that is relevant to this PR is https://github.com/ClusterHQ/flocker/pull/1174/files#diff-bd81b442286984f45ec7fe47f8aa0584R172 which enables the defined service by name, rather than hard-coding the name.

This also allows (although I don't think we should bother documenting) using slightly more complicated commands to only allow access to these services from some addresses.